src/App.tsx

```diff
@@ -118,55 +119,46 @@ class App extends React.Component<any, any> {
     super(props);
     this.state = {
       ...INITIAL_STATE,
-      paymentRequest: this.getPaymentRequest()
+      paymentRequest: this.getPaymentRequest(),
     };
     this.web3Modal = new Web3Modal({
-      network: this.getNetwork(),
+      network: "mainnet",
```

Comment:
Is this correct?

Reply:
Other than that, LGTM and works for the request as described in #3

Reply:
Well I guess here we would need to make some design decisions.... previously there was this concept of chainId that got translated to a network string

In this PR I'm hard-coding to mainnet but I guess we should decide the scope that we want to support.

As a quick thought we could parse the chosen currency from our supported assets and just set this on page load.

Example
if currency = dai  then we set to mainnet
if currency = eth then we set to mainnet
if currency = xdai then we set to xdai

Reply:
It does work for other networks, right? This only means the initial network is set to mainnet. Which might be fine

Reply:
True but I think it's important to set it correctly on page load